### ISSUE TITLE
"If child’s parent is not parent, throw a NotFou..."

### ISSUE THREAD

**Issue by pmkrefeld** [whatwg/dom#98]
https://dom.spec.whatwg.org/#mutation-algorithms

> If child’s parent is not parent, throw a NotFoundError exception. 

No comments needed.
Please provide a specification in proper English.

**Comment by jdm** [comment]
Could you be more specific about the problem?

**Comment by domenic** [comment]
This is proper English. Note how the second "parent" is italicized, referring to a variable.

**Comment by pmkrefeld** [comment]
"parent is not parent" can never be true when written this way (without further specifying what the difference between parent and parent is). A more verbose description would be appreciated.

Your hint: second parent is a variable doesn't help very much, is the first parent not a variable?

**Comment by annevk** [comment]
Did you point those three folks to the specification or let them read it without context and markup? Without context and markup that sentence is indeed hard to parse, but it's not hard to parse in the specification really. When parent is a variable it's italic, when it's a reference it's a hyperlink. It seems relatively clear.

**Comment by annevk** [comment]
Closing this as I don't see the problem.
